src/store/middlewares/offline.js

```diff
@@ -2,14 +2,29 @@ import actionsType from '../../constants/actions';
 import { successToastDisplayed, errorToastDisplayed } from '../../actions/toaster';
 import { loadingStarted, loadingFinished } from '../../utils/loading';
 
+const getErrorMessage = (errorCode, optData) => {
+  switch (errorCode) {
+    case 'EUNAVAILABLE':
+      return `Failed to connect to node ${optData}`;
+    case 'EPARSE':
+      return 'Make sure that you are using the latest version of Lisk Nano.';
+    default: return '';
```

Comment:
There are some differences to the pre-migration logic of this
- The default error message used to be also `Failed to connect to node ${address}. `
- the `EUNAVAILABLE` error message is `Failed to connect: Node ${address} is not active`, not `Failed to connect to node ${address}. `
- the `EPARSE` error message is appended to the default (notice `+=`)

https://github.com/LiskHQ/lisk-nano/blob/bb27b7470cc739390e6f9744e0e46d705272d9a7/src/services/api/peers.js#L118-L126